ocw_data_parser/conftest.py

```diff
@@ -0,0 +1,49 @@
+import os
+import shutil
+import pytest
+import responses
+import boto3
+from moto import mock_s3
+from ocw_data_parser.ocw_data_parser import OCWParser
+
+@pytest.fixture(autouse=True, scope="session")
+def s3_bucket():
+    with mock_s3():
+        conn = boto3.client('s3',
+                            aws_access_key_id="testing",
+                            aws_secret_access_key="testing")
+        conn.create_bucket(Bucket="testing")
+        responses.add_passthru("https://")
+        responses.add_passthru("http://")
+        s3 = boto3.resource('s3',
+                            aws_access_key_id="testing",
+                            aws_secret_access_key="testing")
+        s3_bucket = s3.Bucket(name="testing")
+        yield s3_bucket
+
+@pytest.fixture(autouse=True, scope="function")
+def ocw_parser():
+    """
+    Instantiate an OCWParser object and run functions depending on args passed in
+    """
+    yield OCWParser(course_dir="ocw_data_parser/test_json/course_dir",
+                        destination_dir="ocw_data_parser/test_json/destination_dir",
```

Comment:
This used to use `TemporaryDirectory` but it was reverted in a recent commit. Was that intentional?